src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -144,7 +144,8 @@ function discoverController($scope, config, courier, $route, $window, Notifier,
   $scope.indexPattern = resolveIndexPatternLoading();
   $scope.searchSource
   .set('index', $scope.indexPattern)
-  .highlightAll(true);
+  .highlightAll(true)
+  .version(true);
```

Comment:
Does this need to be added [here](https://github.com/elastic/kibana/blob/2fb2b4ca6ceb6364a6b2f9f7ae36dcf7b948f962/src/core_plugins/kibana/public/dashboard/dashboard.js#L81)? If not, then won't you have a similar issue to what you explained [here](https://github.com/elastic/kibana/pull/10346#issuecomment-279824664)?